projects/storefrontlib/router/app-routing.module.ts

```diff
@@ -9,7 +9,7 @@ import { OnNavigateService } from './on-navigate.service';
     RouterModule.forRoot([], {
       anchorScrolling: 'enabled',
       relativeLinkResolution: 'corrected',
-      initialNavigation: 'enabled',
+      initialNavigation: 'enabledBlocking',
```

Comment:
What is this change about?

Reply:
This is one breaking change of NG14: initialNavigation: 'enabled' was deprecated in v11 and is replaced by initialNavigation: 'enabledBlocking'.